func TestMyFunc(t *testing.T) {
	if 1 == 2 {
		t.Errorf("fail")
	}
}
